scripts/reana_bench.py

```diff
@@ -381,39 +383,78 @@ def _save_original_results(workflow: str, df: pd.DataFrame):
 
 
 def submit(
-    workflow_prefix: str, number_of_workflows: int, file: str, workers: int
+    workflow_prefix: str, workflow_range: (int, int), file: str, workers: int
 ) -> None:
     """Submit multiple workflows, do not start them."""
-    if _workflow_already_exists(workflow_prefix):
```

Comment:
Needed to remove this condition as it will require more coding to check workflow and ranges.

Reply:
Skipping is good, we can give all the liberty to the outside user to submit as they wish.  (And we may pay attention during `collect` in case they submitted twice, e.g. if both `myrun-0.1` and `myrun-0.2` are in the run list...)